packages/teams-js/test/internal/communication.spec.ts

```diff
@@ -1,67 +1,378 @@
 import * as communication from '../../src/internal/communication';
+import { GlobalVars } from '../../src/internal/globalVars';
 import { DOMMessageEvent } from '../../src/internal/interfaces';
 import * as utils from '../../src/internal/utils';
+import { FrameContexts } from '../../src/public';
+import { app } from '../../src/public/app';
+import { Utils } from '../utils';
 
-/* eslint-disable */
-/* As part of enabling eslint on test files, we need to disable eslint checking on the specific files with
-   large numbers of errors. Then, over time, we can fix the errors and reenable eslint on a per file basis. */
+describe('Testing communication', () => {
+  describe('initializeCommunication', () => {
+    describe('frameless', () => {
+      let utils: Utils = new Utils();
```

Comment:
Frameless has its own class called `FramelessPostMocks`, we use this class to test the frameless unit test cases. `Utils` class is used for framed test case.

Reply:
We do use that one a lot, but actually Utils is set up to test both and is a much easier class to work with. FramelessPostMocks for example has a bunch of unit test exceptions baked into it that I don't need or want. Eventually I'd like to get all of the unit tests using Utils for this :)